website/versioned_docs/version-v1.2/cli.md

```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
+| `checkVerifyingKeys` | Check that the verifying keys in the contract match the local ones | `-uq, --use-quadratic-voting"`: Whether to use quadratic voting <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL <br/> `-vk, --vk-contract <vkContract>`: The VkRegistry contract address <br/> `-s, --state-tree-depth <stateTreeDepth>`: The state tree depth <br/> `-i, --int-state-tree-depth <intStateTreeDepth>`: The intermediate state tree depth <br/> `-m, --msg-tree-depth <messageTreeDepth>`: The message tree depth <br/> `-v, --vote-option-tree-depth <voteOptionTreeDepth>`: The vote option tree depth <br/> `-b, --msg-batch-depth <messageBatchDepth>`: The message batch depth <br/> `-p, --process-messages-zkey <processMessagesZkeyPath>`: The process messages zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-t, --tally-votes-zkey <tallyVotesZkeyPath>`: The tally votes zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
there's no `-uq` flag in v1.2?

there's another flag for this command:
`-ss, --subsidy-zkey <subsidyZkeyPath>`
    "the subsidy zkey path (see different options for zkey files to use specific circuits https://maci.pse.dev/docs/trusted-setup, https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)"

we removed subsidy in current version, but there's still subsidy in v1.2